docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -88,37 +91,29 @@ Our guiding principles for this document can be summarized in three points:
 **1. This work is important**
 
 Inclusive language use is important, and we want to begin by acknowledging the 
-potential harm that using gendered language can create. ‘Sex’ and ‘gender’ are 
-not synonymous, unidimensional traits. [Bauer_2022]_ acknowledges this and explains 
-how common sex and gender proxies such as chromosomes or sex organs are imperfect 
-and not fully inclusive of lived experiences.  
-
-Using imperfect proxies and gendered language can create harm by alienating individuals, and by 
-creating invalid research. People who are intersex, transgender, nonbinary, and/or 
+potential harm that using simplistic and gendered language can create.
```

Comment:
I reworked this section, perhaps a bit too much, because I moved some of the content into the intro section below to avoid redundancy.